content/source/docs/cloud/integrations/service-now/example-customizations.html.md

```diff
@@ -45,41 +45,36 @@ This example use case creates a Terraform catalog item for resources that limits
 1. Search for and select the variable set you created (`Example Pinned Variables`).
 1. Click "Submit".
 
-### Create Custom Workflow
+### Create Flow
 
-This example uses the "Example With Pinned Variables" workflow, which is one of the example workflows included with the integration.
+This example uses the "Provision Resources With Vars" flow, which is one of the example flows with actions included with the integration.
 
 In the ServiceNow Studio:
 
-1. Navigate to "Workflow" > "Workflow" > "Example With Pinned Variables".
-1. Double click on the "Run Script" item.
-1. Copy the script code from the editor.
-1. Close the "Run Script" and "Workflow" windows.
-1. Create a new custom workflow, by clicking "Create Application File" and choosing "Workflow".
-    - Name: `Example Pinned Variables` (replace with your custom workflow name)
-    - Table: `Requested Item [sc_req_item]` and Click "Submit"
-1. Add a "Run Script" item to the workflow, by navigating to the "Core" tab and selecting "Utilities" > "Run Script".
-    - Name: `Example Workflow Pinned Variables`
-    - Script:
-        - Paste in the code copied from the "Provision Resources with Variables" workflow, and change the following lines:
-
-            ```javascript
-              // ** Custom Variables **
-              var VAR_SET_NAME = "example_pinned_variables";
-              var VCS_REPO     = "ORG_NAME/REPO_NAME";
-            ```
-
-1. Click "Submit"
-1. Ensure that the ServiceNow Run Script is part of the workflow. (The summary of the workflow should look like "Begin" > "Run Script" > "End".)
-1. Click the menu at the top left and select "Publish"
+1. Navigate to "Flow Designer" > "Flows" > "Provision Resources With Vars".
+   1. This will be flow that can be used as an example.
+2. Create a new flow by selecting "New" in Flow Designer
+3. Name it "Example Flow" and click "submit"
+4. Editing your example flow
+   1. Trigger: Service Catalog
+   2. 1st Action: Get Catalog Variables
+      1. Drag "Requested Item Record" to "Submitted Request" field
+      2. Template Catalog Item: This should be your example catalog item created above
+      3. Catalog Variables: Move all of your variables to "Selected" side
+      4. Select "Done" to finish this action
+   3. Add another action and select the "Terraform Create Workspace with Var"
+      1. Using the Flow "Provision Resources With Vars" as an example:
+      2. Move the variables from the "Data" options on the right of the flow designer to match the example flow
```

Comment:
Should these be indented more (since they are the process described by point 1)?